tracee/tracee.go

```diff
@@ -781,6 +799,38 @@ func (t *Tracee) shouldProcessEvent(e RawEvent) bool {
 
 func (t *Tracee) processEvent(ctx *context, args map[argTag]interface{}) error {
 	switch ctx.EventID {
+
+	//capture written files
+	case VfsWriteEventID, VfsWritevEventID:
+		if t.config.CaptureWrite {
+			filePath, ok := args[t.EncParamName[ctx.EventID%2]["pathname"]].(string)
+			if !ok {
+				return fmt.Errorf("error parsing vfs_write args")
+			}
+			// path should be absolute, except for e.g memfd_create files
+			if filePath == "" || filePath[0] != '/' {
+				return nil
+			}
+			dev, ok := args[t.EncParamName[ctx.EventID%2]["dev"]].(uint64)
```

Comment:
Please have a look at https://github.com/aquasecurity/tracee/blob/main/tracee/consts.go#L834
and https://github.com/aquasecurity/tracee/blob/main/tracee/tracee.go#L1326
This one is of type dev_t, which is uint32

Reply:
Ah, that clears things up much.  Thank you for the direction and bearing with me.